DNN Platform/Website/DesktopModules/Admin/Security/Register.ascx.cs

```diff
@@ -591,12 +591,14 @@ private bool Validate()
                 {
                     CreateStatus = UserCreateStatus.InvalidLastName;
                 }
-
-                var cleanDisplayName = PortalSecurity.Instance.InputFilter(User.DisplayName, filterFlags);
-                if (!cleanDisplayName.Equals(User.DisplayName))
-                {
-                    CreateStatus = UserCreateStatus.InvalidDisplayName;
-                }
+				if (string.IsNullOrEmpty(PortalSettings.Registration.DisplayNameFormat))

```

Comment:
This should be validated if `DisplayNameFormat` is _not_ empty, right?

```suggestion
				if (!string.IsNullOrEmpty(PortalSettings.Registration.DisplayNameFormat))

```

Reply:
No. Only if DisplayNameFormat is empty, DisplayName can be set and has to be validated.
If DisplayNameFormat is set, DisplayName is NULL at that time and set later during user creation.